bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/form/Options.java

```diff
@@ -94,5 +94,26 @@ public String getValue() {
     default Type getType() {
         throw new UnsupportedOperationException();
     }
+    
+    /**
+     * Checks if the this text field is mandatory.
+     *
+     * @return {@code true} if the field must have a input, {@code false} otherwise
+     * @since com.adobe.cq.wcm.core.components.models.form 13.0.0; marked <code>default</code> in 14.1.0
+     */
+    default boolean isRequired() {
```

Comment:
Please update version in https://github.com/adobe/aem-core-wcm-components/blob/development/bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/form/package-info.java#L37 to 14.4.0 for this change